deno/rest/v10/interactions.ts

```diff
@@ -37,7 +37,9 @@ type RESTPostAPIBaseApplicationCommandsJSONBody = AddUndefinedToPossiblyUndefine
 		| 'guild_id'
 		| 'name_localized'
 		| 'description_localized'
-	>
+		| 'default_member_permissions'
+	> &
+		Partial<Pick<APIApplicationCommand, 'default_member_permissions'>>
```

Comment:
I just noticed something: our types are wrong... According to the docs, `default_member_permissions` is possibly missing too 🤔 

CC @almeidx since you PR'd this

Reply:
I guess they haven't updated the docs website yet, because on the repository it is not optional (discord/discord-api-docs@ad81ed31372245d2b4f7006924bd8dd68130a87f)